specs/DotNetProjectFile.Analyzers.Specs/Rules/MS_Build/Ommit_XML_declaration.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace Rules.MS_Build.Ommit_XML_declaration;
```

Comment:
```suggestion
﻿namespace Rules.MS_Build.Omit_XML_declaration;
```
Also in the filename